### PR TITLE
added to can't review

### PR DESCRIPTION
**Pull request by martin-martin**
how to handle submissions according to an outdated rubric; referencing the _Special Cases.md_ document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Please send the following information to review-support@udacity.com.
 * If the submission is complete (contains all necessary files) but doesn't compile:
   * Please tell the student what the issue is.
   * If you are unable to figure out the issue, please be honest with the student. Request the student to reach out to a coach and contact us at review-support@udacity.com.
+* If the project was submitted according to an outdated rubric:
+  * Be sure to include an explanation and the appropriate alternative submission link from the _Special Cases.md_ document.
```
Comment (ccrazy88) [inline README.md]: Can you add a link to it from here? Thanks!

**Comment by martin-martin** [comment]
@ccrazy88 did you see the update with the link? okay like that? : )

**Comment by ccrazy88** [comment]
Yep, thanks for the bump.

**Comment by martin-martin** [comment]
: )

.

**Pull request merged**